Gems/ScriptCanvas/Code/Include/ScriptCanvas/Translation/Configuration.h

```diff
@@ -42,6 +39,9 @@ namespace ScriptCanvas
             AZStd::string_view m_scopeOpen;
             AZStd::string_view m_singleLineComment;
             AZStd::string_view m_suffix;
+            // #scriptcanvas_component_extension
```

Comment:
The changes here reflect what needed to change in the translated Lua files to refer to userdata rather the entity component system directly.